app/models/concerns/status/fetch_replies_concern.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Status::FetchRepliesConcern
+  extend ActiveSupport::Concern
+
+  # debounce fetching all replies to minimize DoS
+  FETCH_REPLIES_DEBOUNCE = 30.minutes
+
```

Comment:
have you thought about putting these (and the MAX_REPLIES below) into a file in config/initializers?
So they can be modified more easily, if necessary for (e.g.) tuning?

Reply:
I was thinking we would either do that or use env vars, ya. I think `config/initializers` is sort of cluttered atm, and sort of "advanced" (most people that are starting up an instance are directed towards editing their `.env.production` first), and using env vars gives handles for people who are running in a prebuilt container/managed host so that's where i was leaning. I don't know ruby patterns that well so if `config/initializers` is the canonical place then we could init from .env there and read it in the class?

Reply:
done in https://github.com/NeuromatchAcademy/mastodon/pull/44/commits/e712283c9a9dbdbe221059b4068c471a0505d15b :)

Reply:
sweet, thanks 👍